### DISCUSSION TITLE
基于 Aptos 的 NFT Marketplace

### DISCUSSION THREAD

**Discussion by zven21** [NonceGeek/Move-Camp#181] [Amazing Things!]
一句话介绍：借（chao）鉴（xi）[Opensea](https://opensea.io/) 做一套基于 Aptos 的 NFT Marketplace

## V0.1

MVP 版本，完成 Collection 和 Token 的内容索引，获取用户的 Token 信息，以及可以正常挂单成单（一口价）

- [x] Collection && Token 索引
- [x] 获取用户的 Token List
- [x] 一口价挂单(listing)
- [x] 取消挂单
- [x] 修改订单价格
- [x] 成单

## V0.2

支持 Auction 模式

- [x] 挂起 Auction
- [x] Bid
- [x] Claim token
- [x] Claim coins

## V0.3

支持 Offer 模式

- [x] Make offer
- [x] Cancel offer
- [x] Accept offer
- [x] Claim offer token

## V0.4

批量 listing, delisting, buy

- [x] Batch listing
- [x] Batch delisting
- [x] Batch buy token
- [x] Batch cancel list

## V0.5

Social 相关

- [ ] 点赞
- [ ] 关注
- [x] Profile

**Comment by leeduckgo** [comment]
专业！

**Comment by zven21** [comment]
* github: https://github.com/myastrallabs/lotus-contract
* project website: https://nftsignal.app/
